### PR TITLE
Protect landing-page sections file

### PR DESCRIPTION
**Pull request by guyfreund**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
We need to protect our landing-page sections file.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Marketplace related
 /Tests/Marketplace/approved_tags.json @demisto/content-leaders @michalgold
 /Tests/Marketplace/approved_usecases.json @demisto/content-leaders @michalgold
+/Tests/Marketplace/landingPage_sections.json @demisto/content-leaders @michalgold
```
Comment (ShahafBenYakir) [inline .github/CODEOWNERS:4]: What is this file I don't see in the repo

Comment (ShahafBenYakir) [inline .github/CODEOWNERS:4]: @Bargenish Can we change the naming to stick with the current convention?

Comment (Bargenish) [reply .github/CODEOWNERS:4]: @ShahafBenYakir  it's a bit different case than the approved tags and usecases so I'm not sure it's right here. but we don't have any limitations to change it.

Comment (ShahafBenYakir) [reply .github/CODEOWNERS:4]: I meant the file name, it is landingPage_sections and should be landing_page_sections

**Pull request merged**